templates/new_device_login.txt

```diff
@@ -1,6 +1,8 @@
 {{t "New sign-in to Firefox" }}
 
 {{ device }}
+{{ ip }}
+{{ location }}
```

Comment:
up to @ryanfeeley, we need to test how the text email looks like. It might not be readable after we add more information to it.